docs/generators/swift5.md

```diff
@@ -53,6 +53,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |swiftUseApiNamespace|Flag to make all the API classes inner-class of {{projectName}}API| |null|
 |useBacktickEscapes|Escape reserved words using backticks (default: false)| |false|
 |useClasses|Use final classes for models instead of structs (default: false)| |false|
+|useCustomDateWithoutTime|Uses a custom type to de- and encode dates without time information to support OpenAPIs date format (default: false)| |false|
```

Comment:
@Jonas1893  Where is says `Uses a custom type to de- and encode` should it bet `Uses a custom type to decode and encode`?